src/bin/tikv-ctl.rs

```diff
@@ -738,6 +770,24 @@ fn main() {
                         .takes_value(true)
                         .help("set the end raw key, in escaped form"),
                 ),
+        )
+        .subcommand(
+            SubCommand::with_name("log-behind")
```

Comment:
When is the command needed? /cc @siddontang

Reply:
When tikv report this error in log file:
```
"raft to_commit {} is out of range [last_index {}]"
```
it's at `raft/raft_log.rs:216`.

Then we can run this command to determine we should transfer-leader or mark the region as tombstone on that peer.

Reply:
Can it be merged with `tombstone` and named it as `recover_region`?

Reply:
This needs more change, including kvproto, pd. I will continue after #2394  merged.

Reply:
If you only want to support transfer leader, you should add a transfer_leader command. 

What here you want is to get a up-to-data follower as the transfee. This should be determined by the user, so I think you can do:

1. Add a command to list all the log index of the peers in the region
2. Choose the best follower
3. Transfer leader